portal/server/utils/csv-utils.js

```diff
@@ -63,6 +65,14 @@ const handleFloatingPointRoundingErrors = (number) => {
   return Number(number.toFixed(FLOATING_POINT_ROUNDING_DECIMAL_PLACES));
 };
 
+/**
+ * Checks if the cell value is a rich text value.
+ * returns true if so as needs further processing to convert to a string.
+ * @param {CellValue} cellValue
+ * @returns {boolean} - true if the cell value is a rich text value.
+ */
+const isRichTextValue = (cellValue) => Boolean(cellValue && Array.isArray(cellValue.richText));
```

Comment:
```suggestion
const isRichTextValue = (cellValue) => cellValue && Array.isArray(cellValue.richText);
```

Reply:
will keep as is as otherwise does not return false if null (returns null)